classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label
+
+  def initialize(label)
+    @label = label
+    @student = []
```

Comment:
- [ ] I think it would be better to name this array "students" because in a classroom you will have multiple students. It's logical and easier to read that way.